fd6130/hsl-bundle/1.0/config/packages/fd_hsl.yaml

```diff
@@ -0,0 +1,19 @@
+fd_hsl:
+    paginator:
+        default_limit: 30
+        
+    # This is image upload listener for VichUploaderBundle.
+    # If you don't want use this listener, just set enable to false.
+    hsl_image_upload_listener:
+        enable: true
+        # Every image you upload will save as the extension you defined.
+        # Please provide extension name without any punctuation.
+        # null mean do nothing.
+        save_as_extension: null
+        quality: 90
+        resize:
+            enable: false
+            # If image larger then this width (px), it will be resize.
+            resize_when_width_exceed: 1024
+            # Resize to width
+            resize_to_width: 1024
```

Comment:
This seams like default config. You dont need to show the user all your config. That is what your documentation is for. 

Please remove these and add a link to your docs. 
Feel free to leve say paginator with the default limit, I guess most users would change that even if 30 is a sensible default.

Reply:
Ok i have remove it.